rag/server.py

```
@@ -0,0 +1,60 @@
+"""FastAPI application for the Local RAG server."""
+
+import textwrap
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from langchain_community.llms import Ollama
```

Pylint (3.3.7)

[error] Unable to import 'langchain_community.llms'

(E0401)